File: src/app/model/emi-detail.ts
import { Emi } from "./emi";
import { LoanSanction } from "./loan-sanction";

export class EmiDetail 
{

    emiId:number;
    // firstName:string;
    // accountNumber:number;
    // emailAddress:string;
    emiTenure:number;
    defaulterCount:number;
    emiPreStatus:string;
    emiPaid:string;
    emiTotalAmount:number;
    emiRoi:number;
    ls:LoanSanction;
    emi:Emi;
   
}

File: SpringBootSmallBusinessApp1/src/main/java/com/ff/SpringBootSmallBusinessApp/app/model/EmiDetail.java
package com.ff.SpringBootSmallBusinessApp.app.model;

import java.util.HashSet;
import java.util.Set;

import javax.persistence.CascadeType;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.JoinColumn;
import javax.persistence.OneToMany;
import javax.persistence.OneToOne;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Entity
@Data
@AllArgsConstructor
@NoArgsConstructor
public class EmiDetail 
{
	@Id
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	private Integer emiId;
	private Integer emiTenure;
	private Integer defaulterCount;
	private String emiPreStatus;
	private Double emiTotalAmount;
	private Double emiRoi;
	@OneToOne(cascade = CascadeType.ALL)
	private LoanSanction ls;
	@OneToMany(cascade = CascadeType.ALL)
	private Set<Emi> emi=new HashSet<>();
	
}